index.html

```diff
@@ -23,25 +23,160 @@
     </header>
 
 
-    <section class="container">
+    <section id="about">
 
-    <h1 class="primary-text">
+    <h1 class="title">
             I’m Miguel Cuello.
             Glad to see you!
-        </h1>
-        <p class="first-text">
+    </h1>
+        <p class="text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-        <h3>LET’S CONNECT</h3>
-        <ul id="social">
-            <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
-            <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
-            <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
-            <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+
+    <p class="project">LET’S CONNECT</p>
+
+    <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
+
+        <img src="images/portfolio.png" alt="a_screentshot" id="portfolio">
+
+        <h2 class="title tonic">Tonic</h2>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
```

Comment:
- [ ] `hrefs` don't require an alt attribute please kindly remove the alt attributes where you have placed the in hrefs in your code base, it is important to stick to the correct HTML standards when designing web pages :+1: